src/main/java/frc/robot/Robot.java

```diff
@@ -31,6 +31,13 @@ public void robotInit() {
     m_robotContainer = new RobotContainer();
     SmartDashboard.putNumber("Test Pivot", ManipulatorConstants.PIVOT_MAX);
     SmartDashboard.putNumber("Shooter Test", 2500);
+    SmartDashboard.putNumber("Turn P val", 0);
+    SmartDashboard.putNumber("Turn I val", 0);
+    SmartDashboard.putNumber("Turn D val", 0);
+    SmartDashboard.putNumber("Turn kS", 0.1);
+    SmartDashboard.putNumber("Turn kV", 0.2);
+    SmartDashboard.putNumber("Turn kA", 0.1);
+    SmartDashboard.putNumber("Shooter Test", 5000);
```

Comment:
should these be removed now that it's working or do you want to keep?
```suggestion
    SmartDashboard.putNumber("Turn P Value", 0);
    SmartDashboard.putNumber("Turn I Value", 0);
    SmartDashboard.putNumber("Turn D Value", 0);
    SmartDashboard.putNumber("Turn kS", 0.1);
    SmartDashboard.putNumber("Turn kV", 0.2);
    SmartDashboard.putNumber("Turn kA", 0.1);
    SmartDashboard.putNumber("Shooter Test", 5000);
```